ngirimana/murikahomeltd
# # 173642315 resolve authentication errors

> #### What does this PR do?
* remove authentication error
> #### Description of Task to be completed
- remove token on signup
- position button appropriately
> #### How should this be manually tested?
* clone repository
* run ``` cd murika-front-end``` in terminal
* checkout  **bg-resolve-authentication-errors-173642315**  branch by running ```git checkout bg-resolve-authentication-errors-173642315``` in terminal
* use 'npm install' to install all necessary dependencies
* Run 'npm start' to run the server 
* And click the login link
> #### Any background context you want to provide?
* you should install react  in your PC 
> #### What are the relevant pivotal tracker stories?
* [#173642315](https://www.pivotaltracker.com/story/show/173642315)
> #### Screenshots (if appropriate)
* N/A
